lib/db/files_db.dart

```diff
@@ -433,7 +458,7 @@ class FilesDB {
           $columnCreationTime=?''',
       whereArgs: [
         title,
-        deviceFolder,
+        deviceFolder ?? "",
```

Comment:
Since we have removed the constraint from this column, we should be able to insert `NULL` values?